otx/recipes/stages/_base_/models/classifiers/classifier.py

```diff
@@ -5,7 +5,6 @@
     task="classification",
     pretrained=None,
     backbone=dict(),
-    neck=dict(type="GlobalAveragePooling"),
```

Comment:
It's just a question. Why is it removed?

Reply:
I have the same question about this, it seems like this could probably affect the backbone replacement.

Reply:
ViT templates are not compatible with GAP, I removed this line. Each `model.py` of current CNN templates already have GAP as their neck. No side effect was observed.